WS2012R2/lisa/xml/CoreTests.xml

```diff
@@ -239,13 +239,19 @@
         <test>
             <testName>CheckNuma</testName>
             <setupScript>
-            	<file>setupScripts\DM_DISABLE.ps1</file>
-            	<file>setupScripts\ChangeCPU.ps1</file>
+                <file>SetupScripts\DM_CONFIGURE_MEMORY.ps1</file>
```

Comment:
Please use the DM_Disable.ps1 script as it has been fixed now to match the requirements.
The parameters required are: vmName, enableDM, staticMem and memWeight.